tvb/simulator/plot/timeseries_interactive.py

```diff
@@ -61,9 +61,11 @@
 import numpy
 import pylab
 import matplotlib.widgets as widgets
+from matplotlib import rcParams
 from tvb.simulator.common import get_logger
 import tvb.datatypes.time_series as time_series_datatypes
 from tvb.basic.neotraits.api import HasTraits, Attr, Int
+from tvb.simulator.plot.utils.data_structures_utils import ensure_list, rotate_n_list_elements
```

Comment:
I see 2 function being used here. Maybe we can copy these alone from under utils, closer to the ts_interactive

Reply:
These 2 functions has been copied directly in the timeseries_interacive.